std/experimental/allocator/gc_allocator.d

```diff
@@ -165,3 +165,17 @@ struct GCAllocator
     // anything above a page is simply rounded up to next page
     assert(GCAllocator.instance.goodAllocSize(4096 * 4 + 1) == 4096 * 5);
 }
+
+@safe unittest
+{
+    import std.typecons : Ternary;
+
+    void[] buffer = GCAllocator.instance.allocate(42);
+    void[] p;
+    Ternary r = GCAllocator.instance.resolveInternalPointer(&buffer[0], p);
+
+    assert(r == Ternary.yes && &p[0] == &buffer[0] && p.length >= buffer.length);
+    assert(GCAllocator.instance.resolveInternalPointer(null, p) == Ternary.no);
+    void *badPtr = (() @trusted => cast(void*)(0xdeadbeef))();
+    assert(GCAllocator.instance.resolveInternalPointer(badPtr, p) == Ternary.no);
```

Comment:
Circle CI fails with:
> std/experimental/allocator/gc_allocator.d(180:56)[warn]: Argument 2 is named 'p', but this is the name of parameter 1

So the course of action is to use a slightly more informative name for the local variables. I would suggest renaming:
* `void[] p` -> `void[] result`
* `Ternary r` -> `Ternary found`

Reply:
Or alternatively, `void[] p` -> `void[] res` and `Ternary r` -> `Ternary ok` if you prefer to be more terse.